src/@types/api.ts

```diff
@@ -0,0 +1,38 @@
+import { Bookmark, Group, Tag } from './domain';
+
+export interface SuccessResponse {
```

Comment:
I remember that response's body should contain status code.

Reply:
Since there's `message` field in response, `statusCode` field might be useless.